lib/jxl/decode_test.cc

```diff
@@ -1685,6 +1685,41 @@ TEST(DecodeTest, PixelTestOpaqueSrgbLossyNoise) {
   }
 }
 
+TEST(DecodeTest, ExtraBytesAfterCompressedStream) {
+  size_t xsize = 123, ysize = 77;
+  size_t num_pixels = xsize * ysize;
+  std::vector<uint8_t> pixels = jxl::test::GetSomeTestImage(xsize, ysize, 3, 0);
+  jxl::CompressParams cparams;
+  CodeStreamBoxFormat box_formats[3] = {
+      kCSBF_None,
+      kCSBF_Single_Zero_Terminated,
+      kCSBF_Multi_Zero_Terminated,
+  };
+  for (int i = 0; i < 3; ++i) {
```

Comment:
Perhaps for-in would be more readable here.

Reply:
Done.